docs/bonus-labs/cluster-autoscaling.md

```diff
@@ -0,0 +1,75 @@
+# Cluster Autoscaling
```

Comment:
Did you purposely avoid linking this Readme from the list of Bonus Labs in the index Readme?

Reply:
Good catch.  Added.